prestoadmin/slider/config.py

```diff
@@ -135,17 +149,23 @@ def write(conf):
 
 
 def get_conf_if_missing():
+    """ Loads the configuration if it hasn't already been loaded. Returns True
+    if the config was loaded for the first time, and False otherwise.
+    """
     with settings(parallel=False):
```

Comment:
How does this work?  You can't set things to be parallel or serial from within a method because at that point it's already been called parallel-ly or serially.  Setting parallel to false will change the attribute, but it doesn't change the behavior.  Since here it first checks if the confs already been loaded, I imagine any issues would be a race condition that you wouldn't necessarily encounter (and it wouldn't matter much if you reloaded the conf several times), but in interactive mode I feel like it could be an issue pretty easily.